tests/utils/v2Data/events.js

```diff
@@ -0,0 +1,310 @@
+const uuid = require('uuid');
+const { UtapiMetric } = require('../../../libV2/models');
+
+const { range, randChoice, randInt } = require('./rand');
+
+const requiredFields = {
+    uuid: () => uuid.v4(),
+    account: params => params.account,
+    location: params => params.location || 'ring',
+};
+
+const objOpFields = {
+    user: params => params.user,
+    bucket: params => params.bucket,
+    object: params => params.object,
+};
+
+const eventTemplates = {
+    get: {
+        ...objOpFields,
+        outgoingBytes: params => params.currentSize,
+    },
+    put: {
+        ...objOpFields,
+        incomingBytes: params => params.newSize,
+        sizeDelta: params => {
+            if (params.versioning) {
+                return params.newSize;
+            }
+            return params.newSize - params.currentSize;
+        },
+        objectDelta: params => {
+            if (params.versioning) {
+                return 1;
+            }
+            return params.exists ? 0 : 1;
+        },
+    },
+    delete: {
+        ...objOpFields,
+        sizeDelta: params => {
+            if (params.versioning) {
+                return 0;
+            }
+            return -1 * Math.floor((params.currentSize / params.versions));
+        },
+        objectDelta: params => (params.versioning ? 0 : -1),
+    },
+    multidelete: {
+        ...objOpFields,
+        sizeDelta: params => {
+            if (params.versioning) {
+                return 0;
+            }
+            return Math.floor((params.currentSize / params.versions)) * -2;
+        },
+        objectDelta: params => (params.versioning ? 0 : -2),
+        object: () => undefined,
+    },
+    md: {
+        user: params => params.user,
+    },
+
+    bucket: {
+        user: params => params.user,
+        bucket: params => params.bucket,
+    },
+    obj: objOpFields,
+};
+
+const eventTemplateToOperation = {
```

Comment:
this feels like something that should be outside of the test utils, but maybe it is only useful for tests not sure.